config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -256,6 +260,7 @@ periodics:
   - "perfDashPrefix: kubemark-100Nodes"
   - "perfDashJobType: performance"
   interval: 3h
+  cluster: scalability
```

Comment:
(picked one kubemark job since it runs more often, on top of the weekly gke jobs)